packages/editor/src/Editor.tsx

```diff
@@ -564,11 +568,24 @@ function calculateViewportRelatedStuff(
         }
       }
     }
+
+    if (viewport.options.height === "max") {
+      height = availableSize.height;
+    }
+
+    if (viewport.options.margin && width < availableSize.width) {
+      const margin = Number(viewport.options.margin);
+
+      if (Number.isFinite(margin)) {
+        height -= margin * 2;
+      }
+    }
```

Comment:
I added additional options for viewport so that it is possible to stretch the device to its full height. Also, an option was added to make margin from the edges of the editor.

Reply:
How do you specify the viewport options (also for the native easyblocks editor topbar?)